controllers/feedback.js

```diff
@@ -0,0 +1,44 @@
+const Feedback = require('../dbfunction/feedbacks');
+
+const FeedbackController = {
+  createNewRecord: async (req, res) => {
+    try {
+      const feedback = req.body;
+      //   check if feedback is an array
+      const isArray = Array.isArray(feedback);
+      if (!isArray) {
+        return res.status(400).send({
+          success: false,
+          message: 'complete the feedback!',
+        });
+      }
+      // check if feedback is complete
+      if (feedback.length !== 20) {
+        return res.status(400).send({
+          success: false,
+          message: 'complete the feedback',
+        });
+      }
```

Comment:
I think you can merge this two validation into one 
what do you think ?  cc: @richard